### PR TITLE
Added Feature to use the Grafana Datasource proxy

### PR DESCRIPTION
**Pull request by langerma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,41 +12,68 @@ Usage:
                             [--duration DURATION]
                             [--function {average,max,min,sum}]
                             [--user USER] [--password PASSWORD]
+                            [--grafana {yes,no}]
```
Comment (andreasjansson) [inline README.md]: update doc to reflect `grafana` being boolean flag

**Comment by langerma** [comment]
i also squashed commit now

File: README.md [hunk 2]
```diff
@@ -12,41 +12,68 @@ Usage:
...
 
     define command {
         command_name    check_graphite
         command_line    $USER1$/check_graphite --host graphite.mydomain:8008 --metric "$ARG1$" --critical $ARG2$ --warning $ARG3$ --user me --password changeme123 2>&1
     }
 
-Example Nagios service:
+    define command {
+        command_name    check_graphite_grafana
+        command_line    $USER1$/check_graphite --host graphite.mydomain:8008 --metric "$ARG1$" --critical $ARG2$ --warning $ARG3$ --grafana yes --datasourceid 23 --authkey 12345ABSCD 2>&1
```
Comment (andreasjansson) [inline README.md]: And maybe change this to `$USER1$/check_graphite --host graphite.mydomain:8008 --metric "$ARG1$" --critical $ARG2$ --warning $ARG3$ --grafana --datasourceid 23 --authkey 12345ABSCD 2>&1`

File: check_graphite [hunk 3]
```diff
@@ -31,8 +32,10 @@ EXIT_WARNING    = 1
 EXIT_CRITICAL   = 2
 EXIT_UNKNOWN    = 3
 
+requests.packages.urllib3.disable_warnings()
```
Comment (andreasjansson) [inline check_graphite:35]: Could we also just add a quick comment on why we need this line?

File: check_graphite [hunk 4]
```diff
@@ -54,9 +57,13 @@ def check(host, metric, warning_threshold, critical_threshold,
         auth = (user, password)
     else:
         auth = None
-
     try:
-        r = requests.get(url, params=params, auth=auth)
+        if grafana:
+            url = str(host) + '/api/datasources/proxy/' + str(datasourceid) + '/render'
```
Comment (andreasjansson) [inline check_graphite]: and one last nitpick: let's make this

``` python
url = '%s/api/datasources/proxy/%s/render' % (host, datasourceid)
```

**Comment by andreasjansson** [comment]
Just a couple more comments, but in general looks good, so thank you very much for this PR!

File: check_graphite (hunk 3 above)
Comment (langerma) [inline check_graphite:35]: i just wanted to surpress the InsecureRequestWarning when i am using a selfsigned SSL Cert

**Pull request merged**